### PR TITLE
Missing dev dependency on 'standard' command

### PR DESCRIPTION
**Pull request by budda**
## Please verify the following:
- [ x] Everything works on iOS/Android
- [ ] `ignite-base` **ava** tests pass
- [ ] `fireDrill.sh` passed

## Describe your PR
Git commits would fail with no standard command as it wasn't getting installed on project init.

### CODE REVIEW

File: ignite-base/package.json.template [hunk 1]
```diff
@@ -62,7 +62,8 @@
     "reactotron-redux": "^1.5.2",
     "reactotron-redux-saga": "^1.5.2",
     "snazzy": "^5.0.0",
-    "standard-flow": "^1.0.0"
+    "standard-flow": "^1.0.0",
```
Comment (GantMan) [inline ignite-base/package.json.template]: did standard flow not work?

Comment (budda) [inline ignite-base/package.json.template]: Ignite default app gave me:

```
$ git commit -m "Re-added swipe cards git url as a dependency"

> giftwishlist@0.0.1 lint /Users/mike/Code/exampleapp
> standard --verbose | snazzy

sh: standard: command not found
```
I couldn't commit anything to Git until I installed 'standard'.

**Comment by budda** [comment]
So no errors in the CI - other than ones outside this patches control regarding fsevents.

**Comment by GantMan** [comment]
go ahead and merge master in for the CI, should be a button on github below this messsage.

**Comment by budda** [comment]
@GantMan "Only those with write access to this repository can merge pull requests." is all I can see.

**Comment by GantMan** [comment]
thanks @budda! Welcome to the Ignite Contributor's list!

**Pull request merged**